Sources/SpelledPitch/PitchSpeller/Wetherfield/SpellingInverter.swift

```diff
@@ -0,0 +1,423 @@
+//
+//  SpellingInverter.swift
+//  SpelledPitch
+//
+//  Created by Benjamin Wetherfield on 2/14/19.
+//
+
+import DataStructures
+import Pitch
+
+struct SpellingInverter {
+    
+    // MARK: - Instance Properties
+    
+    // The pre-assigned `DirectedGraph` which stands in for the `FlowNetwork` that
+    // would solve to the set of spellings passed into the `SpellingInverter`.
+    var flowNetwork: DirectedGraph<PitchSpeller.AssignedNode>
+    
+    let pitchClass: (Int) -> Pitch.Class?
+}
+
+extension SpellingInverter {
+    
+    // MARK: - Type Aliases
+    
+    typealias AssignedEdge = OrderedPair<PitchSpeller.AssignedNode>
+    typealias UnassignedEdge = OrderedPair<PitchSpeller.UnassignedNode>
+    typealias PitchedEdge = UnorderedPair<FlowNode<Cross<Pitch.Class,Tendency>>>
+}
+
+extension SpellingInverter {
+    
+    // MARK: - Initializers
+    
+    init(spellings: [Int: Pitch.Spelling], parsimonyPivot: Pitch.Spelling = .init(.d)) {
```

Comment:
With the added top level static properties on `Pitch.Spelling`, you can now use:

```Swift
init(spellings: [Int: Pitch.Spelling], parsimonyPivot: Pitch.Spelling = .d) {
```

Reply:
whoa